core/entrypoint/2-copy-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ -z "$CHORUS_USER" ]; then
+  echo "CHORUS_USER is not set. Exiting."
+  exit 1
+fi
+
+if [ -z "$CONFIG_ARRAY" ]; then
+  echo "CONFIG_ARRAY is not set. Exiting."
+  exit 0
+fi
+
+# Convert CONFIG_ARRAY string to an array
+CONFIG_ARRAY=( "$CONFIG_ARRAY" )
+
+#copy all configuration files in $CONFIG_ARRAY in $CHORUS_USER homedir
+for CONFIG in "${CONFIG_ARRAY[@]}"; do
```

Comment:
I don't like those bashisms, is that really needed?

Reply:
Unless we change the way to pass the config files yes, suggestions welcome if you have a more convenient way to do it

Reply:
`for CONFIG in ${CONFIG_ARRAY}; do` should do it.

Reply:
it seems so